1_Mechanics/02_GMGuide.md

```diff
@@ -536,30 +586,32 @@ investigation-based with relatively little combat.
 <!-- BROZ: Do we Bystander distinction? Can't an NPC be either Dealer, Minion
  or Companion, and join players in combat accordingly? -->
 
+<!-- LOCK: Bystander to me is our stand-in for "Allied NPC", whereas a Minion is an enemy term and Companion is a Player Controlled character. I don't like the term "NPC" when talking to GMs who are literally playing the characters, so I feel like using Bystander and Minion/Dealer as GM controlled character terms is more useful here. -->
```

Comment:
I'd prefer to have a unified term for non-dealer so that their allegiance is ambiguous 
- A: Are they a dealer?
- B: No, they're a `X`

If `X` could be bystander or minion, saying which gives hints at where the narrative 'should' go.

I also don't love the asymmetry between bystander and minion. The former sounds passive (which, if DM-NPC or allied combatant, they're not) and the latter sounds weak ('yeah, you can defo oneshot this one') when that may not be the case at higher levels

Reply:
What term would you suggest in this case? If we have Dealer (for Player or GM controlled), what is a singular term for non-Dealer? 

Minion has been my term for an Enemy non-dealer, where Bystander was a term we had in there for a long time. Minion won't work for townsfolk or shopkeeps, so we'll want to find that separate term that can be all-encompassing.

I'd like to keep "Minion" as a term for a directly controlled Dealer character (Player or GM) such as the Necromancer, where the Skeletons are "Minions" of them rather than independent thinking enemy characters that are non-dealers.

Reply:
Added as Issue #104